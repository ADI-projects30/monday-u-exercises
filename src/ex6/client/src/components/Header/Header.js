import React from "react";
import { Heading } from "monday-ui-react-core";
import "monday-ui-react-core/dist/main.css";
import "../Header/Header.css";

const Header = () => {
  return (
    <div className="app-name">
      <Heading ellipsis={false} value="TodoApp" />
    </div>
  );
};

export default Header;
