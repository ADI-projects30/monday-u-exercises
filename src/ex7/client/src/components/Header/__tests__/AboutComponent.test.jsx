import renderer from "react-test-renderer";
import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import Header from "../Header"

  
test('renders correctly', () => {
    const tree = renderer.create(
      <Header />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });