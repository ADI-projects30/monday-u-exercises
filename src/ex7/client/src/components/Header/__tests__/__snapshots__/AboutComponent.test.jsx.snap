// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="app-name"
>
  <h1
    aria-label=""
    className="heading-component element-type-h1 size-large"
    id=""
  >
    TodoApp
  </h1>
</div>
`;
